amelia.html

```
@@ -0,0 +1,614 @@
+
+      <!-- Moves library -->
+      <h2 id="moves" class="text-2xl font-semibold mb-2 scroll-mt-20">
+        <span id="moves-heading">20 concrete moves</span>
+      </h2>
+      <p class="text-slate-400 text-sm mb-4">
+        Filter by which affirmation you want to practice. Pick 1–3 for the week.
+        Don't try all of them — that's not essentialist.
+      </p>
+
+      <div id="filters" class="flex flex-wrap gap-2 mb-5"></div>
+      <div id="moves" class="grid md:grid-cols-2 gap-3 mb-5"></div>
```

HTMLHint (1.9.2)

[error] The id value [ moves ] must be unique.

(id-unique)